packages/web-react/src/components/Icon/Icon.tsx

```diff
@@ -20,7 +20,7 @@ export const Icon = ({ name, title, size, ariaHidden, ...restProps }: IconProps)
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore: Incompatible HTMLElement and SVGSVGElement
     <svg
-      viewBox={`0 0 ${size} ${size}`}
```

Comment:
In case someone wants to continue [the discussion](https://github.com/lmc-eu/spirit-design-system/pull/510#issuecomment-1264738624).
Despite that, I would use the default props variable, because it currently corresponds with the viewport icon size, but I don't have a strong opinion on it.